rnn/rnn.c

```diff
@@ -1167,7 +1167,7 @@ static void preptypeinfo(struct rnndb *db, struct rnntypeinfo *ti, char *prefix,
 			ti->type = RNN_TTYPE_A3XX_REGID;
 		} else {
 			ti->type = RNN_TTYPE_HEX;
-			fprintf (stderr, "%s: unknown type %s\n", prefix, ti->name);
+			//fprintf (stderr, "%s: unknown type %s\n", prefix, ti->name);
```

Comment:
oh, I hadn't noticed these amongst the rest of the stderr spam.. but I think this one is useful to keep (after filtering out the special address/waddress types)

Reply:
Maybe like the new push?  not sure.

Reply:
yeah, lgtm.. r-b